README.md

```diff
@@ -22,22 +22,107 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+Add an instance of GraphQL::Analyzer to your schema, instantiate it with a list of instrumentation to capture different datastore queries.
```

Comment:
list of instrumentation**s**

Reply:
Maybe would be helpful to explicitly list the instrumentations that are already implemented?